tutorials/ten_minutes.ipynb

```diff
@@ -10,9 +10,9 @@
     "\n",
```

Comment:
<p>Why is it "little" code change here? I think there was no code change?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fugue-project/tutorials/pull/150/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>I think stating that there is no code change is a bit of a stretch and not entirely accurate. We still had to:</p><p>1) convert the input_df to a spark df</p><p>2) Wrap the function in transform() and specify arguments.</p><p> </p><p>I get what you mean though. We had to make no changes to the underlying Pandas based predict function, which is awesome!</p><p> </p><p>I just think you guys need to be a bit careful with the wording as you don't want to hype up expectations too high for users but I'm happy to resort back to the original commentary also.</p>

Reply:
<p>Ah crap. I just noticed that this commentary was referring to the predict function solely. I'll revert back to 'no modifications'.</p>

Reply:
Yes no change for the underlying Pandas function. We are a minimal layer on top. We can always be more clear on claims!